### ISSUE TITLE
[BUG] Chart showMin/Max-Value overlapping  axes

### ISSUE THREAD

**Issue by kodeaben** [kirbydesign/designsystem#2591]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
 

Whenever the `showMinValue` or `showMaxValue` is set and this value is close to one of the axes, it might overlap the axis.

## Describe how to reproduce the bug
Edit the dataset in `_dataset` in `apps/cookbook/src/app/examples/charts-example/examples/stock-chart-example.component.ts:53`

Example 1: minValue overlaps x-axis and y-axis: 
``` typescript 
  _dataset = [
    {
      data: [
        { x: 1637049659000, y: 12 },
        { x: 0, y: 1110 },
      ],
      borderColor: getThemeColorHexString('secondary'),
    },
  ];
```
![Screenshot 2022-11-08 at 10 09 57](https://user-images.githubusercontent.com/114976818/200523152-fae0d2dc-4b45-452c-8a7f-89d3778387dd.png)

Example 2: maxValue is hidden behind the edge of the chart:

``` typescript
  _dataset = [
    {
      data: [
        { x: 0, y: 20 },
        { x: 2, y: 2 },
        { x: 3, y: 6 },
      ],
      borderColor: getThemeColorHexString('secondary'),
    },
  ];
```
![Screenshot 2022-11-08 at 14 34 30](https://user-images.githubusercontent.com/114976818/200578406-93ebf5bc-f6c9-4913-8787-2243b80037eb.png)

Navigate to http://localhost:4200/#/home/showcase/chart-stock.
The Chart should now be shown with the issue

## What was the expected behavior?

Expected behaviour is that the labels does not overlap.
## Are there any additional context?
The label mustn´t overlap the graph line (see comment)

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [x] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by MartinCBBankdata** [comment]
Yes, The only thing I would like to add is the solution does not put the label across the graph line. See the picture below. 
![image](https://user-images.githubusercontent.com/110449604/200570395-3bac6edf-729a-464c-927e-b8d44aca4ada.png)

**Comment by kodeaben** [comment]
Possible solution. This is the same data as in Example 1
![Screenshot 2022-11-09 at 11 00 22](https://user-images.githubusercontent.com/114976818/200800550-a7a5f036-2a1a-4686-b629-fe26ef48f740.png)

This is the same data as in Example 2
![Screenshot 2022-11-09 at 11 02 03](https://user-images.githubusercontent.com/114976818/200800903-6491969c-8b99-492e-9dbf-e1830541eabf.png)

Same examples with a larger y-value
Example1: 
![Screenshot 2022-11-09 at 11 04 38](https://user-images.githubusercontent.com/114976818/200802048-2400242e-30d7-47f8-baa6-9e3c04fce37b.png)

Example2:
![Screenshot 2022-11-09 at 11 09 00](https://user-images.githubusercontent.com/114976818/200802358-badb4618-bacd-42ca-b5ce-700723ddbf83.png)

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
